src/listeners/ui.listener.ts

```diff
@@ -2,12 +2,9 @@ import { disableMenu } from '../utils/menu'
 import { openTab } from '../utils/tabs'
 
 function initialize() {
```

Comment:
not used function, also above not used import `disableMenu`

Reply:
I think this listener can be completely removed

Reply:
Yes, actually the whole file is not needed any more. But I left it because it might be needed in future. But it's simple so it makes sense to delete it and return it back later if needed.